aip/general/0191.md

```diff
@@ -48,6 +48,9 @@ file per service.
 APIs with only one file **should** use a filename corresponding to the name of
 the API.
 
+API `service` definitions and associated RPC request and response `message`
+definitions **must** be defined in the same file.
```

Comment:
I wonder whether should would be more appropriate than must here. I would be surprised if any tooling actually relied on this, so having a "must" for something that is basically for the sake of readability feels slightly excessive. On the other hand, I'd be surprised if this actually caused a problem for many APIs.

@jgeewax any thoughts?

Reply:
I'm torn. I can see both arguments. That being said, I don't see any hard reason why it **must** be true, so I would lean towards **should**.

Reply:
I'm similarly on the fence.  I landed on `must` in the interest of consistency; later on, when specifying intra-file structure, the AIP mandates that request/response messages `must` be defined together and ordered so it seems strange to convey varying degrees of obligation for these two related points.

> The RPC request and response message definitions, in the same order of the
> corresponding methods. Each request message must precede its
> corresponding response message (if any).

Should we similarly weaken the language above as well?

That said, as @jgeewax points out `should` can easily be promoted to `must` at some later point if this causes more confusion than it eliminates.

Reply:
Sounds like the consensus is on `should` until we find a reason to be more strict and change to `must` ? If so, let's change that and get this merged.

Reply:
All set.